save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -176,11 +186,4 @@ class WarnPlugin(
             else -> Fail("")
         }
     }
-
-    private fun testFileName(): String = "test_file${atomicInt.addAndGet(1)}"
```

Comment:
I thought, that such unique name was created in aim to avoid overwriting in multi thread execution.
What we decide to do now?

Reply:
Yes, but turns out we need to preserve the original file name. We also have an issue to preserve full path (#126), so after that these copied files will be unique automatically.